### ISSUE TITLE
ion-tab-button in angular no longer registers double clicks to navigate with talkback turned on.

### ISSUE THREAD

**Issue by Koramur** [ionic-team/ionic-framework#30627]
### Prerequisites

- [x] I have read the [Contributing Guidelines](https://github.com/ionic-team/ionic-framework/blob/main/docs/CONTRIBUTING.md#creating-an-issue).
- [x] I agree to follow the [Code of Conduct](https://ionicframework.com/code-of-conduct).
- [x] I have searched for [existing issues](https://github.com/ionic-team/ionic-framework/issues) that already report this problem, without success.

### Ionic Framework Version

v8.x

### Current Behavior

In angular Ionic ion-tab-button it no longer works with Talkback turned. This can be replicated just using ionic's sample on the website: https://ionicframework.com/docs/api/tabs

https://stackblitz.com/run?file=src%2Fapp%2Fexample.component.html

With talkback turned on it reads "double click to navigate" but double clicking with talkback on does nothing. This does work with Jaws, NVDA and Voice Over, as well with no screen reader turned on, it just does not work with talkback turned on.

### Expected Behavior

Talkback should read "double click to navigate" and then double clicking should navigate to the view.

### Steps to Reproduce

1. navigate to url with tabs
2. turn on talkback on android device
3. navigate through page until you get to the tabs
4. try to double click on a tab to navigate

### Code Reproduction URL

https://stackblitz.com/run?file=src%2Fapp%2Fexample.component.html

### Ionic Info

I dont know how to get this, it says its not available.

### Additional Information

I believe this also occurs in lower versions of ionic, I just noticed it in version 8. It seems like talkback was updated and now it does not work?

**Comment by theissitfc-maker** [comment]
We are having the same issue. We fixed it by adding the href property. We are using the tabs with routing, so that was no problem for us.

Example:
```   
<ion-tab-button tab="profile" href="/profile">
    <ion-icon aria-hidden="true" name="person-outline"></ion-icon>
    <ion-label role="navigation">Profil</ion-label>
</ion-tab-button>
```
